Python/bytecodes.c

```diff
@@ -2009,7 +2010,7 @@ dummy_func(
             // cancel out the decrement that will happen in LOAD_SUPER_ATTR; we
             // don't want to specialize instrumented instructions
             PAUSE_ADAPTIVE_COUNTER(this_instr[1].counter);
-            GO_TO_INSTRUCTION(LOAD_SUPER_ATTR);
+            GO_TO_INSTRUCTION(LOAD_SUPER_ATTR, INLINE_CACHE_ENTRIES_LOAD_SUPER_ATTR);
```

Comment:
```suggestion
            GO_TO_INSTRUCTION(LOAD_SUPER_ATTR);
```
The code generator knows how big the instruction is.